ui/src/components/Description/index.tsx

```diff
@@ -41,6 +69,8 @@ const Description: React.FC<Props> = (props) => {
                       <Tab eventKey={0} title="Description" id={props.name}>
                         <hr className="hub-horizontal-line"></hr>
                         <ReactMarkDown
+                          linkTarget={' '}
```

Comment:
Will this be blank, if yes do we need it ??

Reply:
yes, it requires for opening the url link into a new tab